source/linux/Overview/_Processor_SDK_Building_The_SDK.rst

```diff
@@ -116,24 +116,24 @@ The MACHINE can be set to |__SDK_BUILD_MACHINE__|, for example.
 
             $ git clone https://git.ti.com/git/arago-project/oe-layersetup.git tisdk
             $ cd tisdk
-            $ ./oe-layertool-setup.sh -f configs/processor-sdk/processor-sdk-08.06.00-config.txt
+            $ ./oe-layertool-setup.sh -f configs/processor-sdk-analytics/processor-sdk-analytics-10.01.00-config.txt
             $ cd build
             $ . conf/setenv
-            $ git clone https://git.ti.com/git/security-development-tools/core-secdev-k3.git -b master
-            $ export TI_SECURE_DEV_PKG_K3=`pwd`/core-secdev-k3
+            $ echo 'ARAGO_BRAND = "edgeai"' >> conf/local.conf
             $ MACHINE=am62axx-evm bitbake -k tisdk-edgeai-image
 
+         Your tisdk-edgeai-image wic image will be generated in arago-tmp-[toolchain]/deploy directory. Use `Processor\_SDK\_Linux\_create\_SD\_card <Overview/Processor_SDK_Linux_create_SD_card.html>`__ to flash this image on the SD-Card.
 
       .. ifconfig:: CONFIG_part_variant not in ('AM62AX')
 
          The final command below will build the :file:`tisdk-default-image`, which is the
          Processor SDK image with arago filesystem.  See `Build Options`_ for a list of
          additional targets.
-         
+
          .. tabs::
-            
+
```

Comment:
Is this addition of new line intentional?

Reply:
No

Reply:
I was going to say this is just removing the trailing whitespace. I'd prefer if you keep that change.